Classes/Domain/Search/SearchRequest.php

```diff
@@ -339,6 +340,24 @@ public function getHasFacetValue(string $facetName, $facetValue): bool
         return $this->activeFacetContainer->hasFacetValue($facetName, $facetValue);
     }
 
+    /**
+     * Returns all sortings in the sorting string e.g. ['title' => 'asc', 'relevance' => 'desc']
+     *
+     * @return array
+     */
+    public function getSeperatedSortings(): array
+    {
+        $parsedSortings = [];
+        $explodedSortings = GeneralUtility::trimExplode(',', $this->getSorting());
```

Comment:
The third parameter, `removeEmptyValues`, should be set, otherwise, further code will stumble over non-existent array keys.

Reply:
Good point. I added another commit.